0000-lts-releases.md

```diff
@@ -0,0 +1,54 @@
+- Start Date: 2022-04-20
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+The core team would like to move to an LTS release strategy
+
+# Motivation
+[motivation]: #motivation
+
+Our curret release strategy causes us to provide security and bugfixes for as
+many as five versions. Our documented [life cycle](https://saltproject.io/salt-platform-support/) has become less
+sustainable as we've increased the frequency of releasing new versions of Salt.
+
+# Design
+[design]: #detailed-design
+
+We would like to move to maintaining at most two branches/versions of Salt. We
+can accomplish this by designating a single version as Long Term Support (LTS).
+A LTS version would be supported with bugfixes and security fixes for a longer
+period of time than our current life cycle allows. I propose a 2 year LTS window.
+
+
+## Alternatives
+[alternatives]: #alternatives
+
+Change the current life cycle to provide support for less versions and/or for a shorter durration per version.
+
+## Unresolved questions
+[unresolved]: #unresolved-questions
+
+- One big question is how long does an LTS version remain supported? I propose a
+2 year support cycle but I could see a case made for 3 years.
+
+- At what point do we stop supporting OSes that have reached end-of-life status for the LTS version?
+
+@dmurphy18: I would propose that we stop support as now for OS's which have EOL'd on the LTS Release,
+- reasoning being if the OS is not providing FREE security fixes, we should not
+  either. PAID support for security fixes is another matter and between
+  customer and VMware, not the Salt Project.
+- non-LTS branch - does it get security fixes or do they come in the next
```

Comment:
what specifically is meant by non-lts branch here? are we suggesting that builds of the "main" branch might not get a re-release at the same time as the "LTS" branch if a cve came out?

What about users who want to be on "latest"

Reply:
I have added a diagram to hopefully show what we'll be doing better. Security releases are a priority. When we encounter a CVE all supported releases get a patch for the CVE at the same time, when we make the CVE public.